src/clr-angular/utils/component/is-boolean-attribute-set.spec.ts

```diff
@@ -0,0 +1,21 @@
+/*
+* Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+* This software is released under MIT license.
+* The full license information can be found in LICENSE in the root directory of this project.
+*/
+import { isBooleanAttributeSet } from './is-boolean-attribute-set';
+
+describe('isAttributeSet', () => {
+  it('should return boolean true', () => {
+    expect(isBooleanAttributeSet('')).toBeTruthy();
+    expect(isBooleanAttributeSet(true)).toBeTruthy();
+    expect(isBooleanAttributeSet('true')).toBeTruthy();
+    expect(isBooleanAttributeSet('random string ')).toBeTruthy();
+  });
+
+  it('should return boolean false', () => {
+    expect(isBooleanAttributeSet('false')).toBeFalsy();
```

Comment:
@coryrylan Do we want this? This is a special case that isn't really to spec.

Reply:
This should be truthy not falsy if the string is `'false'`, we don't want to diverge from the HTML attribute spec as this will become more important with custom elements in the future. Example here: https://stackblitz.com/edit/angular-shr5nb